packages/discovery-react-components/src/components/ResultsPagination/__tests__/ResultsPagination.test.tsx

```diff
@@ -116,62 +116,6 @@ describe('ResultsPaginationComponent', () => {
         pageSize: 20
       });
     });
-
-    test('will add pageSize as a pageSize selection if it is not already included', async () => {
```

Comment:
Why remove this ability? It seems odd that it was previously supported, but keeping it seems simple enough and prevents a breaking change.

Reply:
We needed this due to the mismatch of `results_per_page` and `count` values being sent in from the shim, but no longer need it. This functionality also caused some strange behaviors. For example, if the user adjusts the page size from a value that doesn't exist in the default list to one that does exist, the value is then removed from the list, which is strange behavior from the user's perspective (i.e. the dropdown menu options seem to change arbitrarily).